replica/table.cc

```diff
@@ -549,8 +549,17 @@ void table::enable_off_strategy_trigger() {
 
 storage_group_manager::~storage_group_manager() = default;
 
+storage_group* storage_group_manager::storage_group_for_id(const schema_ptr& s, size_t i) const {
+    auto it = storage_groups().find(i);
+    if (it == storage_groups().end()) [[unlikely]] {
+        on_fatal_internal_error(tlogger, format("Storage wasn't found for tablet {} of table {}.{}", i, s->ks_name(), s->cf_name()));
```

Comment:
Why fatal? We can continue with an exception in case we have such a bug.

Reply:
agreed. I was thinking about it yesterday, plain exception or on_internal_error is enough?